src/components/pages/chart/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box } from "@chakra-ui/react";
+import { generateChartData } from "@utils/chart";
+import { cleanAggregationData } from "@utils/field";
+import React, { useMemo } from "react";
+
+import { ChartHeader } from "./header";
+import Stats from "./stats";
+
+export default function ChartComponent({ chartData }) {
+  const statsData = useMemo(() => {
+    const aggregationData = cleanAggregationData(chartData.stats.aggregation);
+    return generateChartData(aggregationData, chartData.filtersList);
+  }, [chartData]);
+
+  const dataForChart = Object.entries(chartData.stats.satewiseAggregation)
+    .map(([state, count]) => ({
+      Name: state.toUpperCase(),
+      Value: count
+    }))
+    .filter((item) => item.Name !== "?")
+    .sort((a, b) => (b.Value as number) - (a.Value as number));
+
+  const stateData = dataForChart.length > 0 && {
+    Title: "Statewise distribution",
```

Comment:
should we move this to translation file?

Reply:
yes it can have translation.